cmd/cancel.go

```diff
@@ -63,18 +63,20 @@ func newCancelCmd() *cobra.Command {
 			}
 
 			// Ensure the user really wants to do this.
-			prompt := fmt.Sprintf("This will irreversibly cancel the currently running update for '%s'!", s.Name())
-			if !yes && !confirmPrompt(prompt, s.Name().String(), opts) {
+			stackName := string(s.Ref().Name())
+			prompt := fmt.Sprintf("This will irreversibly cancel the currently running update for '%s'!", stackName)
+			if !yes && !confirmPrompt(prompt, stackName, opts) {
 				return errors.New("confirmation declined")
 			}
 
 			// Cancel the update.
-			if err := backend.CancelCurrentUpdate(commandContext(), s.Name()); err != nil {
+			if err := backend.CancelCurrentUpdate(commandContext(), s.Ref()); err != nil {
```

Comment:
Nit: i find .Re nigh unreadable.  It's totally unclear to me what that is actually returning, or why it would be the right thing to pass in here.  Using 's' for 'stack' is also pretty bad.  When i see code like this i have to go "what is 's'?  what is s' Ref?  Why is that the right thing to pass in here".  Basically, abbreviated c-style naming just makes readability and reviews harder :)